framework/src/eventloop/eventloop_loop.c

```diff
@@ -0,0 +1,54 @@
+/****************************************************************************
+ *
+ * Copyright 2018 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+#include <stdio.h>
+#include <debug.h>
+#include <libtuv/uv.h>
+#include <eventloop/eventloop.h>
+
+#include "eventloop_internal.h"
+
+uv_loop_t *_loop;
+
+uv_loop_t *get_loop(void)
+{
+	if (_loop == NULL) {
+		_loop = uv_default_loop();
+	}
+	return _loop;
+}
+
+void eventloop_loop_run(void)
+{
+	uv_run(get_loop(), UV_RUN_DEFAULT);
+}
+
+int eventloop_loop_quit(void)
```

Comment:
The api name needs to be reconsidered here.
Pairwise naming convention might be comfortable for developers like:
start/stop, run/stop, open/close, etc.